### PR TITLE
Finish user edit, update, index, and destroy actions

### PR DESCRIPTION
**Pull request by piiknight**

### CODE REVIEW

**Comment by piiknight** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -21,10 +29,43 @@ def create
...
+    flash[:danger] = t "login_page.please_login"
+    redirect_to login_url
+  end
+
+  def correct_user
+    @user = User.find_by id: params[:id]
+    redirect_to(root_url) unless current_user?(@user)
+  end
+
+  def admin_user
```
Comment (tranducquoc) [inline app/controllers/users_controller.rb]: 3 hàm logged_in_user, correct_user, admin_user move vào application controller

**Comment by tranducquoc** [comment]
commented

**Comment by piiknight** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**